common/api/core-backend.api.md

```diff
@@ -2128,6 +2174,8 @@ export class ExternalSource extends InformationReferenceElement {
     constructor(props: ExternalSourceProps, iModel: IModelDb);
     // @internal (undocumented)
     static get className(): string;
+    // (undocumented)
+    protected collectReferenceConcreteIds(referenceIds: EntityReferenceSet): void;
```

Comment:
beta function uses alpha type in signature. please find and fix all occurrences.

Reply:
`collectReferenceConcreteIds` is internal, maybe I screwed up extract-api? rerunning

Reply:
looks like I need to mark every override of it internal

Reply:
done